src/components/Comment.js

```diff
@@ -11,11 +33,9 @@ class Comment extends Component {
         </figure>
         <div className="media-content">
           <div className="content">
-            <p>
-              <strong>{this.props.comment.name}</strong>
-              <br />
-              {this.props.comment.comment}
-            </p>
+            <span className="userName">{this.userName}</span>
```

Comment:
there is two different naming conventions for classnames here. 
Stick to dashes for class names unless there is a specific reason for this?

Reply:
There is no specific reason, I just didn't notice I used different naming convention during the development. I'll use dashes instead.

Reply:
Since this change affects the file created in step 10, I reverted code to Step 9 and re-created all following steps in order to reflect the change. If this fix was just another commit, code in step 10,15 and 16 would differ from what is written in the tutorial. This way I ensured the code is same as it is in the tutorial.